wayfinder/manifest.json

```diff
@@ -0,0 +1,50 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "a68bad83-ba55-4350-a913-2f98bb667bad",
+  "app_id": "wayfinder",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Send Wayfinder metrics to Datadog",
+    "title": "wayfinder",
+    "media": [],
+    "classifier_tags": [
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS"
+    ]
+  },
+  "assets": {
+    "integration": {
+      "source_type_name": "wayfinder",
+      "configuration": {
+        "spec": "assets/configuration/spec.yaml"
+      },
+      "events": {
+        "creates_events": false
+      },
+      "metrics": {
+        "prefix": "wayfinder.",
+        "check": "wayfinder.workqueue.depth",
+        "metadata_path": "metadata.csv"
+      },
+      "service_checks": {
+        "metadata_path": "assets/service_checks.json"
+      }
+    },
+    "dashboards": {
+      "Wayfinder": "assets/dashboards/wayfinder_overview.json"
+    },
+    "logs": {}
+  },
+  "author": {
+    "support_email": "jonny.scott@appvia.io",
```

Comment:
For support, we ask that you please use an email distribution list rather than an individual's email

Reply:
Amended